client/src/includes/initDismissibles.ts

```diff
@@ -1,36 +1,55 @@
-import { WAGTAIL_CONFIG } from '../config/wagtailConfig';
-
-export function updateDismissibles(data: Record<string, boolean>) {
-  return fetch(WAGTAIL_CONFIG.ADMIN_URLS.DISMISSIBLES, {
+import { WAGTAIL_CONFIG } from '../config/wagtailConfig'
```

Comment:
Check you have run the formatter I think there should be a line break here 

`npm run format`